### DISCUSSION TITLE
How can I debug juicefs on the GoLand? I want to learn the logic of the underlying operation of juicefs

### DISCUSSION THREAD

**Discussion by T-TRz879** [juicedata/juicefs#2090] [Q&A] [answered]
I use the make command to generate juicefs. I use 「sudo ./juicefs format localhost test 」initializes the disk, and then I configure the command in Golan as shown in the screenshot. Reasonably speaking, it should be possible to debug the write and read processes of juicefs, but I can't
![goland_pic](https://user-images.githubusercontent.com/50860504/169967225-e47455f8-94c4-4b61-b2e7-bce862e35223.png)
![debug_log](https://user-images.githubusercontent.com/50860504/169967294-0b9fb88d-93d3-4e3b-8c56-7ec90eaf30ce.png)

**Comment by xiaogaozi** [comment, marked as answer]
There's a related discussion on Slack: https://juicefs.slack.com/archives/C01KUQ63A8J/p1653375471586789

**Reply by T-TRz879** [reply]
thanks for you ：）
